cnd/src/http_api/swaps.rs

```diff
@@ -69,6 +70,7 @@ async fn handle_get_swap(
     id: LocalSwapId,
     storage: Storage,
     connectors: Connectors,
+    bitcoin_fee_rate_per_vbyte: ::bitcoin::Amount,
```

Comment:
Maybe we could shorten this variable to something like this? The `per` already gives away that it is a rate.

```suggestion
    btc_per_vbyte: ::bitcoin::Amount,
```